backend/app/evaluation/README.md

```
@@ -0,0 +1,402 @@
+### Setup
+
+1. Ensure `GUARDRAILS_API_URL` is set in your `.env` file (see `.env.example`). Optionally set `GUARDRAILS_TIMEOUT_SECONDS` (default: `60`).
+2. Ensure the API is running and accessible at the configured URL.
+
+### Run
+
+```bash
+python3 app/evaluation/multiple_validators/run.py --auth_token <your-token>
+```
+
+The `--auth_token` argument is the plain-text bearer token (without the `Bearer ` prefix).
```

markdownlint-cli2 (0.22.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)